black_it/search_space.py

```diff
@@ -63,8 +63,8 @@ def __init__(
         SearchSpace._check_bounds(parameters_bounds, parameters_precision)
 
         # The bounds we were given are well formed. Save them.
-        self._parameters_bounds = parameters_bounds
-        self._parameters_precision = parameters_precision
+        self._parameters_bounds = np.array(parameters_bounds)
+        self._parameters_precision = np.array(parameters_precision)
```

Comment:
This snippet is the gist of change 9bbf0453b5bf.

The code base is full of external conversions to `nd.array()`, that are now superfluous, for example: https://github.com/bancaditalia/black-it/blob/bbdacdc698a7f87231e31a34684f0f3fc59fea1c/tests/test_calibrator.py#L48-L49

This could be an occasion for simplifying this aspect.

Reply:
Solved by #6, merged before this PR.